App/pages/Projects/containers/SectionMenu.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import uuid from 'react-native-uuid'
+
+import Menu from '../components/Menu'
+import {
+  getCurrentProjectUuid,
+  addVersion,
+  copyVersion,
+  copySection
+} from '../../../state/projects'
+
+export default ({ index, versionUuid, sectionUuid }) => {
+  const projectUuid = useSelector(getCurrentProjectUuid)
+  const newVersionUuid = `version.${uuid.v4()}`
+  const newSectionUuid = `section.${uuid.v4()}`
+  const dispatch = useDispatch()
+  const copyVersionHandler = () => {
+    dispatch(copyVersion({ projectUuid, sectionUuid, versionUuid, newVersionUuid }))
+  }
+  const addVersionHandler = () => {
+    dispatch(addVersion({ projectUuid, sectionUuid, versionUuid: newVersionUuid }))
+  }
+  const copySectionHandler = () => {
+    dispatch(copySection({ projectUuid, sectionUuid, index, newSectionUuid }))
+  }
+  return <Menu
```

Comment:
can be done in a nicer way
like: 
pass single prop to menu component called "options"
and make options to look like - 
const options = [
  { label: 'label1', clickHandler: () => {} },
  { label: 'label2', clickHandler: () => {} }
]

so inside menu component you can simple make a map over this array and create required view without passing specific click handlers